drivers/modem/ublox-sara-r4.c

```diff
@@ -1312,15 +1444,15 @@ static void modem_reset(void)
 	}
 
 	/* stop functionality */
-	ret = send_at_cmd(NULL, "AT+CFUN=0", MDM_CMD_TIMEOUT);
+	ret = send_at_cmd(NULL, "AT+CFUN=0", MDM_REGISTRATION_TIMEOUT);
 	if (ret < 0) {
 		LOG_ERR("AT+CFUN=0 ret:%d", ret);
 		goto error;
 	}
 
 #if defined(CONFIG_BOARD_PARTICLE_BORON)
 	/* use external SIM */
-	ret = send_at_cmd(NULL, "AT+UGPIOC=23,0,0", MDM_CMD_TIMEOUT);
+	ret = send_at_cmd(NULL, "AT+UGPIOC=23,0,0", K_SECONDS(10));
```

Comment:
I increased the MDM_CMD_TIMEOUT to 10 seconds rather than hard coding, not sure if that would have an inpact on the entire functionality.